pkg/command/backup/cmd.go

```diff
@@ -25,18 +25,19 @@ type command struct {
 	flag.TaskBase
 	client *managerclient.Client
 
-	cluster          string
-	dc               []string
-	location         []string
-	keyspace         []string
-	retention        int
-	retentionDays    int
-	rateLimit        []string
-	snapshotParallel []string
-	uploadParallel   []string
-	dryRun           bool
-	showTables       bool
-	purgeOnly        bool
+	cluster             string
+	dc                  []string
+	location            []string
+	keyspace            []string
+	retention           int
+	retentionDays       int
+	rateLimit           []string
+	snapshotParallel    []string
+	uploadParallel      []string
+	dryRun              bool
+	showTables          bool
+	purgeOnly           bool
+	manifestParallelism int
```

Comment:
Naming: note snapshotParallel and uploadParallel, we should use common naming scheme.